handbook/people-ops/onboarding/index.md

```diff
@@ -18,6 +18,50 @@ At Sourcegraph, we don't think of teammates as resources — we think of them as
 
 Never assume that a problem is somebody else's to fix!
 
+## Onboarding standards and success
+
+### Onboarding expectations
+This is a basic breakdown of our expectations for providing a smooth onboarding process:
+
+- New hire has the tools and resources they need to navigate their day to day (accounts, software...)
+- New hire has the necessary support from their peers, manager, and People team (regular 1-1s, shadowing...)
+- New hire knows where to find answers to their questions 
+- New hire experiences no logistical challenges (payroll, expenses, benefits, setting up accounts or other systems...)
+- The onboarding checklist is manageable and easy to navigate by new hire
+
+### Onboarding experience
+We will be putting surveys in place to collect feedback from all new teammates to make sure that their onboarding experience is properly equipping them with the support and resources necessary for them to be successful in their new role. 
+
+#### Onboarding content
+Survey questions:
```

Comment:
Maybe we could say "tentative" or "possible" survey questions

Reply:
In general, one principle of the handbook (which I cannot find in the handbook now haha...) is that it should represent what we're already doing, not what we aspire to do. Whatever our current survey includes is what we should put here, and we can update as needed in the future!

Reply:
I'll update the wording to reflect that we're doing it, though we still don't have anyone that started this FY and has passed the 3 month mark (so it's still technically in the future).